src/ApiInterface/include/ApiInterface/Orbit.h

```diff
@@ -369,16 +369,20 @@ inline bool orbit_api_active() {
 }  // extern "C"
 
 // Internal macros.
-#define ORBIT_CONCAT_IND(x, y) (x##y)
+#define ORBIT_CONCAT_IND(x, y) x##y
 #define ORBIT_CONCAT(x, y) ORBIT_CONCAT_IND(x, y)
 #define ORBIT_UNIQUE(x) ORBIT_CONCAT(x, __COUNTER__)
 #define ORBIT_VAR ORBIT_UNIQUE(ORB)
 
+#define ORBIT_SCOPE_WITH_COLOR_AND_GROUP_ID_INTERNAL(name, col, group_id, pc_name) \
+  uint64_t pc_name;                                                                \
```

Comment:
I am a bit worried to use a fixed variable name here. What if this is take (unlikely), or simply someone is adding two "Scopes" in the same scope?

Reply:
It's not a fixed name, the name is passed as parameter of the macro and is generated with `ORBIT_VAR`.

Reply:
nit. I'm guessing using ORBIT_VAR directly here instead of passing it as parameter doesn't work?

Reply:
The thing is that I need to reference that name in the two lines below, so I need a fixed name before entering the macro.